examples/run.py

```diff
@@ -1,21 +1,64 @@
-import sys
+import argparse
 
 from cleanvision.imagelab import Imagelab
 
 if __name__ == "__main__":
-    dataset_path = sys.argv[1]
+    parser = argparse.ArgumentParser(description="Demonstrates how to use Imagelab")
+    parser.add_argument("--path", type=str, help="path to dataset", required=True)
+    args = parser.parse_args()
 
-    # Run Imagelab with default settings
-    imagelab = Imagelab(dataset_path)
-    imagelab.list_default_issue_types()
-    imagelab.find_issues()
+    dataset_path = args.path
+
+    """
+    Example 1
+
+    This example demonstrates using Imagelab to
+    1. Explore the dataset
+    2. Find all types of issues
+    3. Report top issues found and visualize them
+    4. Check Imagelab attributes: imagelab.issue_summary, imagelab.issues, imagelab.info
+    """
+
+    imagelab = Imagelab(dataset_path)  # initalize imagelab
+    imagelab.list_default_issue_types()  # list default checks
+    imagelab.visualize()  # visualize the dataset
+
+    imagelab.find_issues()  # Find issues in the dataset
     imagelab.report()
 
-    # Run Imagelab for specific checks
+    print("Summary of all issues checks\n", imagelab.issue_summary.to_markdown())
+    imagelab.visualize(
+        issue_types=["blurry"], examples_per_issue=8
+    )  # visualize specific issues
+
+    # Get all images with blurry issue type
+    image_filepaths = imagelab.issues[
+        imagelab.issues["blurry_bool"] == True
+    ].index.to_list()
+    imagelab.visualize(
+        filepaths=image_filepaths
+    )  # visualize images with given filepaths
```

Comment:
```suggestion
    blurry_images = imagelab.issues[
        imagelab.issues["blurry_bool"] == True
    ].index.to_list()
    imagelab.visualize(
        image_paths=blurry_images
    )  # visualize the given images files
```

Reply:
[nit] recommend changing arg name: `filepath` -> `image_path` to make it clearer these are individual images

Reply:
[nit] I recommend changing argument name `filepaths` -> `image_files` which I think is less ambiguous